### PR TITLE
Add sign in link and get started page to header navigation

### PR DESCRIPTION
**Pull request by jamie-o-wilkinson**
### What problem does this pull request solve?

We want users who visit our product page to be able to sign up to GOV.UK Forms for a trial account, or sign in to their existing account.

This PR adds a link to sign in to forms-admin to the header navigation, and also directs new users to the new get started page (added in #116) from the home page call to action.

Trello card: https://trello.com/c/xebzfFm9, https://trello.com/c/4p6qkzvc

<details><summary>Screenshot</summary>
<p>

Screenshot of GOV.UK Forms product page, with Get Started and Sign in links in header, and updated call to action

</p>
</details> 

### Things to consider when reviewing

- Does it work when run on your machine?
- Is it clear what the code is doing?
- Do the commit messages explain why the changes were made?
- Are there all the unit tests needed?
- Has all relevant documentation been updated?

### CODE REVIEW

File: config/settings.yml
Comment (lfdebrux) [inline config/settings.yml:1]: I'd like to use this change in my PR for the get started page; can we split it out into it's own PR to merge now?

File: config/settings.yml [hunk 1]
```diff
@@ -16,3 +16,7 @@ zendesk:
   defaults:
     group_id: 5455184128796 # 1st Line--GOV.UK Forms
     organization_id: 21891972 # GDS
+
+forms_admin:
+  # URL to form-admin
+  base_url: http://localhost:3000
```
Comment (lfdebrux) [inline config/settings.yml:22]: I'd like to use this change in my PR for the get started page; can we split it out into it's own PR to merge now?

Comment (lfdebrux) [reply config/settings.yml:22]: I've done this in https://github.com/alphagov/forms-product-page/pull/113

Comment (jamie-o-wilkinson) [reply config/settings.yml:26]: Woops I see this is duplicated at the top of the file...

**Comment by lfdebrux** [comment]
I've pushed changes to add a link to the get started page as well, for ticket https://trello.com/c/4p6qkzvc, so we can release both these related changes at once.

Updated screenshot:

image

**Review by thomasiles — APPROVED** [review]
